<launch>

<!-- Default folder with maps-->
<arg name="folder" default="$(env HOME)/demo"/>

<!-- Distance Measurement Node -->
<node pkg="stroll_bearnav" type="odometry_monitor_node" name="odometry_monitor"/>

<!-- feature extraction -->
<node pkg="stroll_bearnav" type="feature_extraction_node" name="feature_extraction">
<remap from="image" to="/stereo/joined/image_raw"/>
</node>

<node pkg="stroll_bearnav" type="mapper_node" name="mapper" output="screen">
<remap from="image" to="/stereo/joined/image_raw"/>
<param name="folder" value="$(arg folder)" />
<param name="angularSpeed" value="1.1" />
</node>

<!-- map loader node -->
<node pkg="stroll_bearnav" type="map_preprocessor_node" name="map_preprocessor">
<param name="folder" value="$(arg folder)" />
</node>

<!-- navigator node -->
<node pkg="stroll_bearnav" type="navigator_node" name="navigator" output="screen">
<param name="folder" value="$(arg folder)" />
</node>

</launch>
